README.md

```diff
@@ -1,2 +1,74 @@
-# hydrogen
- 
+<h1 align="center" style="color: #1c64f2; font-weight: bold">HYDROGEN</h1>
+
+<p align="center">A Component Library</p>
+
+---
+
+## Installation Guide
+
+You can use the Hydrogen library by adding our stylesheet to your website. To get started add the following `link` element in your `head` element of the HTML file.
+
+```html
+<link rel="stylesheet" href="https://hydrogen-ui.netlify.app/style.css" />
+```
+
+Visit [docs](https://hydrogen-ui.netlify.app/) to learn more.
```

Comment:
Suggestion: directly link the docs URL instead of the homepage.

Reply:
Fixed!